blog/serializers/post_serializers.py

```
@@ -15,8 +19,15 @@ class PostDetailSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = Post
+        fields = []
```

Ruff (0.15.5)

[warning] Mutable default value for class attribute

(RUF012)